coding/compressed_bit_vector.cpp

```diff
@@ -312,6 +312,28 @@ unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromBitGroups(
   return make_unique<SparseCBV>(setBits);
 }
 
+// static
+unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromCBV(CompressedBitVector const & cbv)
+{
+  auto strat = cbv.GetStorageStrategy();
+  switch (strat)
+  {
+  case CompressedBitVector::StorageStrategy::Dense:
+  {
+    DenseCBV const & dense = static_cast<DenseCBV const &>(cbv);
+    auto bitGroups = dense.m_bitGroups;
+    return CompressedBitVectorBuilder::FromBitGroups(move(bitGroups));
```

Comment:
How about to implement FromBitGroups(vector const & bitGroups), like FromBitPositions? In this case you can replace this code by
```c++
return CompressedBitVectorBuilder::FromBitGroups(dense.m_bitGroups);
```
?